src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -311,6 +318,36 @@ function plant_absorbed_pfd(
     return FT(SW_IN * F_abs)
 end
 
+"""
+    diffuse_fraction(td::FT, T::FT, SW_IN::FT, UR::FT, θs::FT) where {FT}
+
+Computes the fraction of diffuse radiation (`diff_frac`) as a function
+of the solar zenith angle (`θs`), the total surface incident shortwave radiation (`SW_IN`),
+the air temperature (`T`), the relative humidity (`UR`), and the day of the year
+(`td`).
```

Comment:
Could we add a reference for where this formula comes from? also, `UR` in doc string still.

Reply:
Gu, L., Baldocchi, D., Verma, S. B., Black, T. A., Vesala, T., Falge, E. M., & Dowty, P. R. (2002). Advantages of diffuse radiation for terrestrial ecosystem productivity. Journal of Geophysical Research: Atmospheres, 107(D6), ACL 2-1. https://doi.org/10.1029/2001JD001242

Braghiere, R. K. (2013). Avaliação dos fluxos turbulentos de CO2, calor sensível e latente em função da profundidade óptica do aerossol sobre o arco do desflorestamento da Amazônia Legal Brasileira. https://doi.org/10.11606/D.14.2013.tde-29082013-165325